polymetis/polymetis/python/torchcontrol/models/torchscript_pinocchio.py

```diff
@@ -32,13 +32,36 @@ class RobotModelPinocchio(torch.nn.Module):
     Implemented as a ``torch.nn.Module`` wrapped around a C++ custom class that leverages
     `Pinocchio <https://github.com/stack-of-tasks/pinocchio>`_ -
     a C++ rigid body dynamics library.
+
+    Args:
+        urdf_filename (str): path to the urdf file.
+        ee_link_name (str, optional): name of the end-effector link. Defaults to None,
+                                      which would require you to specify link_name
+                                      when using methods which require a link frame;
+                                      otherwise, the end-effector link will be used
+                                      by default.
     """
 
-    def __init__(self, urdf_filename: str, ee_link_name: str):
+    def __init__(self, urdf_filename: str, ee_link_name: str = None):
         super().__init__()
         self.model = torch.classes.torchscript_pinocchio.RobotModelPinocchio(
-            urdf_filename, ee_link_name
+            urdf_filename, False
         )
+        self.ee_link_name = ee_link_name
+        if self.ee_link_name is not None:
+            self.ee_link_idx = self.model.get_link_idx_from_name(self.ee_link_name)
+        else:
+            self.ee_link_idx = None
```

Comment:
We should have a have a `set_ee_link(link_name)` method so that the link name can be set dynamically, and update the error messange in `_get_link_idx_or_use_ee` to inform the user of this additional way to set the ee link.